GPU/Directx9/VertexShaderGeneratorDX9.cpp

```diff
@@ -265,7 +265,7 @@ void GenerateVertexShaderDX9(int prim, char *buffer, bool useHWTransform) {
 			WRITE(p, "  float2 v_texcoord: TEXCOORD0;\n");
 	}
 	WRITE(p, "  float4 v_color0    : COLOR0;\n");
-	if (lmode) 
+	if (lmode && !throughmode)
```

Comment:
Well, there was one case where it affected color0... does lmode never do anything during throughmode?  I just tried to keep the logic the same.

-[Unknown]